aipl/ops/csv.py

```diff
@@ -0,0 +1,12 @@
+from aipl import defop, Table, Column
+
+# assumes header row
+@defop('csv-parse', None, 1.5)
+def op_csv_load(aipl, fname:str) -> dict:
+    import csv
+    ret = []
+    with open(fname, newline='') as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            ret.append(row)
```

Comment:
In general let's `yield row` instead of composing the entire list before returning it.  It's immaterial now but there are designs on streaming and yielding will make streaming that much easier.